misc/git-hooks/pre-commit

```diff
@@ -199,14 +199,16 @@ function check_typos() {
 }
 export -f check_typos
 
-parallel ::: \
+parallel \
+  --jobs 16 \
```

Comment:
I take it that these tasks are mostly io bound and so using more processes than cores is ok?

Reply:
I think so, yes.